### DISCUSSION TITLE
Why is the Rust code intellisense in Tauri so laggy and slow?

### DISCUSSION THREAD

**Discussion by ruinmi** [tauri-apps/tauri#11736] [Q&A] [answered]
I'm using RustRover.

Here is the video:
https://imgur.com/kPEEwdD

**Comment by FabianLars** [comment, marked as answer]
tauri apps are quite large for rust projects with many dependencies. the use of macros probably doesn't help.
i assume rustrover just can't handle projects like this yet. i can't think of anything _we_ can do here right now because we can't just remove dependencies or features.
